packages/mongo-async/oplog_tests.js

```diff
@@ -18,152 +21,152 @@ Tinytest.addAsync("mongo-livedata - oplog - cursorSupported", async function (te
-//         docs.push({
-//           name: "cat " + i,
-//           species: 'cat',
-//           color: 'red'
-//         });
-//       }
-//       for (i = 0; i < self.RELEVANT_SIZE; ++i) {
-//         docs.push({
-//           name: "dog " + i,
-//           species: 'dog',
-//           color: 'blue'
-//         });
-//       }
-//       // XXX implement bulk insert #1255
-//       var rawCollection = self.collection.rawCollection();
-//       rawCollection.insertMany(docs, Meteor.bindEnvironment(expect(function (err) {
-//         test.isFalse(err);
-//       })));
-//     },
-//
-//     function (test, expect) {
-//       var self = this;
-//
-//       test.equal(self.collection.find().count(),
-//                  self.IRRELEVANT_SIZE + self.RELEVANT_SIZE);
-//
-//       var blueDog5Id = null;
-//       var gotSpot = false;
-//
-//       // Watch for blue dogs.
-//       const gotSpotPromise = new Promise(resolve => {
-//         self.subHandle = self.collection.find({
-//           species: 'dog',
-//           color: 'blue',
-//         }).observeChanges({
-//           added(id, fields) {
-//             if (fields.name === 'dog 5') {
-//               blueDog5Id = id;
-//             }
-//           },
-//           changed(id, fields) {
-//             if (EJSON.equals(id, blueDog5Id) &&
-//                 fields.name === 'spot') {
-//               gotSpot = true;
-//               resolve();
-//             }
-//           },
-//         });
-//       });
-//
-//       test.isTrue(self.subHandle._multiplexer._observeDriver._usesOplog);
-//       test.isTrue(blueDog5Id);
-//       test.isFalse(gotSpot);
-//
-//       self.skipped = false;
-//       self.skipHandle = MongoInternals.defaultRemoteCollectionDriver()
-//         .mongo._oplogHandle.onSkippedEntries(function () {
-//           self.skipped = true;
-//         });
-//
-//       // Dye all the cats blue. This adds lots of oplog mentries that look like
-//       // they might in theory be relevant (since they say "something you didn't
-//       // know about is now blue", and who knows, maybe it's a dog) which puts
-//       // the OplogObserveDriver into FETCHING mode, which performs poorly.
-//       self.collection.update({species: 'cat'},
-//                              {$set: {color: 'blue'}},
-//                              {multi: true});
-//       self.collection.update(blueDog5Id, {$set: {name: 'spot'}});
-//
-//       // We ought to see the spot change soon!
-//       return gotSpotPromise;
-//     },
-//
-//     function (test, expect) {
-//       var self = this;
-//       test.isTrue(self.skipped);
-//
-//       //This gets the TOO_FAR_BEHIND back to its initial value
-//       MongoInternals.defaultRemoteCollectionDriver()
-//         .mongo._oplogHandle._resetTooFarBehind();
-//
-//       self.skipHandle.stop();
-//       self.subHandle.stop();
-//       self.collection.remove({});
-//     }
-//   ]
-// );
+process.env.MONGO_OPLOG_URL && testAsyncMulti(
```

Comment:
Do you know why this test was commented? 

Is it ok to run it again?

Reply:
It was here  [b8f2c071190c829975d22ef71f8a1f086c7bb75f](https://github.com/meteor/meteor/commit/b8f2c071190c829975d22ef71f8a1f086c7bb75f)
It is okay. all tests for collection and oplog are green

Reply:
Awesome :+1: